index.js

```diff
@@ -44,8 +44,18 @@ function uninstallDebugContextMenu(win) {
 	win.webContents.removeListener('devtools-closed', deactivateDebugContextMenu);
 }
 
-module.exports = () => {
+module.exports = opts => {
+	opts = opts || {
+		toggle: false
+	};
```

Comment:
my bad. I didn't change the name of the option. just remove it? we need a default value for opts. like this
```
opts = opts || {
        showDevTools: false
};
```